frameworks/zcomet.zsh

```diff
@@ -0,0 +1,36 @@
+() {
+  local -r home_dir=${1}
+
+  git clone https://github.com/agkozak/zcomet.git $home_dir
+
+  # add modules to .zshrc
+  >| ${home_dir}/.zshrc <<\END
+source ${HOME}/zcomet.zsh
+zcomet load zimfw/environment
+zcomet load zimfw/git
+zcomet load zimfw/input
+zcomet load zimfw/termtitle
+zcomet load zimfw/utility
+zcomet load zimfw/duration-info
+zcomet load zimfw/git-info
+zcomet load zimfw/asciiship
+zcomet fpath zsh-users/zsh-completions src
+zcomet load zsh-users/zsh-autosuggestions
+zcomet load zsh-users/zsh-syntax-highlighting
+zcomet load zsh-users/zsh-history-substring-search
+# zcomet adds functions to fpath but does not autoload them!
+() {
+  setopt LOCAL_OPTIONS EXTENDED_GLOB
+  local zfunction
+  for zfunction in ${HOME}/.zcomet/repos/zimfw/*/functions/^(*~|*.zwc(|.old)|_*|prompt_*_setup)(N-.:t); do
+    autoload -Uz ${zfunction}
+  done
+}
```

Comment:
Cool that zcomet is now compatible with prezto-format modules! This means we can remove this code, right?
```suggestion
```

Reply:
Excellent!